app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -314,8 +314,10 @@ def parse_endpoint(entity)
       new_result
     end
 
-    def parse_namespaces(container_projects)
-      parse_base_item(container_projects).except(:namespace)
+    def parse_namespace(namespace)
+      new_result = parse_base_item(namespace).except(:namespace)
+      new_result[:labels] = parse_labels(namespace)
+      new_result
```

Comment:
That's what I had (modulo line breaks) before Ari suggested the current var.
(See comment on outdated diff https://github.com/ManageIQ/manageiq/pull/7771#discussion_r58850231)
Being a ruby newbie, I'm happy to be told which way to code, but please agree between you :-)